src/Classes/LinkRenderers/LibraryLinkRenderer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace LaraZeus\Sky\Classes\LinkRenderers;
+
+use Illuminate\Database\Eloquent\Model;
+use LaraZeus\Sky\Models\Library;
+use LaraZeus\Sky\SkyPlugin;
+
+class LibraryLinkRenderer extends NavLinkRenderer
+{
+    public static string $renders = 'library-link';
+
+    public function getModel(): ?Model
```

Comment:
Similar to previous comment but inverted - if we split the abstract class into 2 tiers then nullable could be removed.